.github/workflows/publish_docs.yaml

```diff
@@ -0,0 +1,54 @@
+name: Publish docs
+
+on:
+  push:
+    branches:
+      - main
+      - v*.*
+
+env:
+  ELIXIR_OPTS: "--warnings-as-errors"
+  ERLC_OPTS: "warnings_as_errors"
+  LANG: C.UTF-8
+  AWS_S3_BUCKET: "s3.hex.pm"
+
+jobs:
+  release_pre_built:
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - otp: 24
+            otp_version: 24.3
+          - otp: 25
+            otp_version: 25.0
+            build_docs: build_docs
```

Comment:
In theory we only need otp: 25 to publish the docs but I am thinking we could also use this action to push Precompiled.zip to S3. So I propose to indeed keep things as is and rename this action to be Publish to repo.hex.pm?

Reply:
Actually, I just realized: wouldn't it be better to move this matrix to the `release_pre_built.yml`, if possible? Both publishing actions will always use the same OTP and Ubuntu versions, so keeping it in one place is fewer things to change?

Reply:
What if we download prebuilt from hex pm s3 bucket instead? We might not need this matrix anymore.

Reply:
The goal is to replace hex.pm s3 buckets with this PR in the long term. :)

Reply:
Ahh i see.